runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/PackageUtil.java

```diff
@@ -119,6 +120,10 @@ static PackageAttributes createPackageAttributes(File source,
       return new PackageAttributes(size, hash, directory, target, source.getPath());
     } catch (IOException e) {
       throw new RuntimeException("Package setup failure for " + source, e);
+    } finally {
+      if (countingOutputStream != null) {
```

Comment:
Hmm. I had thought that with the closing at L109 we didn't need this, but I see that we still might.

Maybe instead just make a `Closer` up front and let it do all the work for you?

https://github.com/google/guava/wiki/ClosingResourcesExplained